src/main/GameOfLife.java

```diff
@@ -52,7 +53,7 @@ private GameBoard generate(GameBoard gameBoard) {
 
     private Cell processStrategiesForCell(int rowNumber, int colNumber, GameBoard gameBoard) {
         Cell processedCell = new Cell();
-        processedCell.setProcessed(false);
+        List<GameOfLifeStrategy> strategies = strategyManager.getStrategies();
```

Comment:
The strategies should be executed into the StrategyManager because is his responsibility. In your case, you are using the class only as a wrapper of the strategies collection which is a good practice in other contexts.